src/libinjector/linux/linux_utils.h

```diff
@@ -151,11 +151,12 @@ struct injector
     // KEEP THESE IN TOP and in sync with the order in injector_utils.c
     injector_step_t step;
     bool step_override; // set this as true for jumping to some arbitrary step
-    bool set_gprs_only;
+    bool dummy; // set_gprs_only
```

Comment:
Doesn't seem to be used anywhere

Reply:
This is due to this comment:
![image](https://user-images.githubusercontent.com/20656851/178689716-c5905f7e-3fa2-4f67-91dc-5760ce3dc452.png)

This is used here https://github.com/tklengyel/drakvuf/blob/582ca8b1175a5aa56f640a0dbdb73b7d60b23732/src/libinjector/injector_utils.c#L116 and here https://github.com/tklengyel/drakvuf/blob/582ca8b1175a5aa56f640a0dbdb73b7d60b23732/src/libinjector/injector_utils.c#L142